### ISSUE TITLE
Is there a way to span a grid item for all available empty grid slots?

### ISSUE THREAD

**Issue by kizu** [rachelandrew/cssgrid-ama#68]
I'm trying to achieve a two-column layout with sidenotes, here is an example: https://codepen.io/kizu/pen/MmbLLG

The problem I have is that when there are multiple grid items placed in one column, I'd want the grid item in the previous column to span all those available spaces. In the example above I achieve this by hacking the `grid-row` with the `span 99` value, which does the exact thing I want. But it surely is a hack and I wonder if there is a way to tell those items to span to all available spaces without the hack.

In other words, I wish there'd be a way to tell a grid item not to create grid tracks outside of its column/row, that could be really helpful in a number of ways (I looked at other questions and I think #60, for example, could be also possible via something like that). If there is no such thing planned and if you'd like, I can elaborate on this idea about limiting the tracks creation.

**Comment by rachelandrew** [comment]
If you just want to span to the final line of the explicit grid, use -1 as the value. Otherwise I'm afraid I don't understand the question.
